tooltips.js

```diff
@@ -0,0 +1,27 @@
+
+function setChildVisibility(element, visibility) {
+    const child = element.firstElementChild
+    if (child != null)
+        child.style.visibility = visibility
+}
+
+const clickButton = document.getElementById("click-button")
+clickButton.onclick = () => {
```

Comment:
The better way to set event listener is:
```javascript
clickButton.addEventListener('click', function () {
   ...
})
```
Using this method you can assign any count of listeners and remove listener by `clickButton.removeEventListener`